src/applications/mhv-landing-page/selectors/mhvAccountStatus.js

```diff
@@ -11,6 +11,17 @@ export const mhvAccountStatusUsersuccess = state => {
   );
 };
 
+export const mhvAccountStatusErrorsSorted = state => {
+  if (state?.myHealth?.accountStatus?.data?.errors) {
+    return state?.myHealth?.accountStatus?.data?.errors.sort(
+      (a, b) =>
+        userActionErrorCodes.includes(b.code) -
+        userActionErrorCodes.includes(a.code),
+    );
+  }
+  return [];
+};
```

Comment:
Can you talk more about what you want to do here?
From manual local testing with this mock data (assuming that we can get multiple error codes):
```
[{"title":"The server responded with status 422","detail":"things fall apart","code":"802"},{"title":"The server responded with status 500","detail":"things fall apart","code":"500"},{"title":"The server responded with status 422","detail":"things fall apart","code":"805"},{"title":"The server responded with status 422","detail":"things fall apart","code":"801"}]
```
The output of mhvAccountStatusErrorsSorted is:
```
[{"title":"The server responded with status 422","detail":"things fall apart","code":"805"},{"title":"The server responded with status 422","detail":"things fall apart","code":"801"},{"title":"The server responded with status 422","detail":"things fall apart","code":"802"},{"title":"The server responded with status 500","detail":"things fall apart","code":"500"}]
```
Is this the expected output?

Reply:
Actually, maybe we cannot get a mix of error status 422 & 500?

Reply:
The idea is to prioritize user actionable errors. If the api somehow returns multiple errors, some of which are user actionable and some of which are not, I consider the user errors to be the most important thing to display. So I sort the errors based on if they are in the user-actionable list and I display the top of the array.

Reply:
Copying in my slack comment:

> Maybe in the refactor we can rename the variable and/or function to indicate the kind of sorting/prioritization (or add a code comment)

There may be a few ways to clarify the purpose of this code for our future selves.

Something for the next iteration, perhaps?

Reply:
`mhvAccountErrorsActionablePriority` is the closest I can get right now, and that's clunky

Reply:
Based on the above output of mhvAccountStatusErrorsSorted mock data, the rendered error code for alert & card need to be updated to use the same data.
<img width="836" alt="Screenshot 2024-12-19 at 3 48 35 PM" src="https://github.com/user-attachments/assets/06728d6c-a4c6-4dc7-93a1-63d6cf1c90af" />